### PR TITLE
Update documentation for Roots 3.1.0 release.

### PR DESCRIPTION
**Pull request by noobanidus**
I can't work out how to build it to test it. :/

### CODE REVIEW

File: docs.json [hunk 1]
```diff
@@ -1126,15 +1126,33 @@
       "Roots 3": {
         "Animal Harvest Ritual": "Mods/Roots_3/animalharvest.md",
         "Bark Creation": "Mods/Roots_3/bark.md",
+        "Chrysopoeia Spell": "Mods/Roots_3/chrysopoeia",
```
Comment (jaredlll08) [inline docs.json:1129]:
```suggestion
        "Chrysopoeia Spell": "Mods/Roots_3/chrysopoeia.md",
```

File: docs/Mods/Roots_3/pyre.md [hunk 2]
```diff
@@ -53,14 +56,14 @@ Pyre.removeRecipe(<roots:stalicripe>);
...
+Pyre.addRecipe("stalicripe2", <roots:stalicripe>*64, [<minecraft:diamond_block>, <minecraft:gold_block>, <minecraft:iron_block>, <minecraft:emerald_block>, <minecraft:deadbush>], 30);
 ```
 
 ### Notes
 
-It's extremely important when replacing recipes to ensure that the recipe name is the same to make certain that Patchouli correctly reports the correct recipe for crafting base items.
+It's extremely important when replacing recipes to ensure that the recipe name is the same to make certain that Patchouli correctly reports the correct recipe for crafting base thaumcraft.items.
 
-For all other items, please use a name descriptive of what your recipe does.
+For all other thaumcraft.items, please use a name descriptive of what your recipe does.
```
Comment (jaredlll08) [inline docs/Mods/Roots_3/pyre.md:69]: thaumcraft?

File: docs/Mods/Roots_3/runicshears.md [hunk 3]
```diff
@@ -62,4 +94,4 @@ RunicShears.removeRecipe(<roots:fey_leather>);
 
 ### Notes
 
-Note that the `removeRecipe` function will attempt to remove any recipe (both runic shearing of blocks and of entities) that matches the desired output.
+Note that the `removeRecipe` function will attempt to remove any recipe (both runic shearing of blocks and of thaumcraft.entities) that matches the desired output.
```
Comment (jaredlll08) [inline docs/Mods/Roots_3/runicshears.md:97]: thaumcraft?

File: docs/Mods/Roots_3/summoncreatures.md [hunk 4]
```diff
@@ -65,7 +65,7 @@ import mods.roots.SummonCreatures;
 // the Animal Harvest map
 SummonCreatures.clearLifeEssence();
 
-// Add a recipe to summon a Chicken using 3 items
+// Add a recipe to summon a Chicken using 3 thaumcraft.items
```
Comment (jaredlll08) [inline docs/Mods/Roots_3/summoncreatures.md:68]: thaumcraft?

**Comment by noobanidus** [comment]
ARGH. I hate how that happened.

**Comment by noobanidus** [comment]
When I was removing some Thaumcraft stuff, IDEA decided to rename all uses of "items" in strings wiht "thaumcraft.items". I guess I missed some of them. :(

**Pull request merged**